### PR TITLE
Implement mobile menu

### PR DESCRIPTION
**Pull request by Akorede3133**
In this PR, the following was effected:
* Designed the menu for mobile
* Created Javascript functions to handle when to display mobile menu

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @Akorede3133, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the student's success team and they'll permit you to work solo or look for a suitable partner.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const headerHeroSection = document.querySelector('.header-hero-section');
+hamburgerIcon.addEventListener('click', () => {
+  navLink.classList.add('show-link');
+  headerHeroSection.classList.add('blur-header-hero-section');
+});
+
+closeIcon.addEventListener('click', () => {
+  navLink.classList.remove('show-link');
+  headerHeroSection.classList.remove('blur-header-hero-section');
+});
```
Comment (Zuhaib042) [inline index.js]: - [ ] Kindly consider implementing the feature where when user clicks on any menu link, the correct part of the page appears. 
requirement:
 -  When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Akorede3133 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Professional Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Akorede3133 @CrackedYoda 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Created mobile menu properly ✔️ 
- Mobile menu design is identical to that of the Figma template ✔️ 
- Links in the mobile menu direct users to the correct part of the page ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Akorede3133** [comment]
Merge main with menu

**Pull request merged**